web/src/lib/domain-logic/shouldShowDisclaimerForProfileNotSubmittingData.ts

```diff
@@ -19,6 +19,6 @@ export const shouldShowDisclaimerForProfileNotSubmittingData = (
     isFormedOutsideNavigator(business) ||
     businessPersona === "OWNING" ||
```

Comment:
```suggestion
    isOwningBusiness(business)
```
Opportunity to utilize the refactor?

Reply:
I can fix this one, I was planning to create a chore to update them across the app

Reply:
I've made additional updates in the commit, but there is additional work to be done https://www.pivotaltracker.com/story/show/186826739